### DISCUSSION TITLE
Can't stop/restart time tracking

### DISCUSSION THREAD

**Discussion by Tertuliano** [kimai/kimai#3015] [On-Premise: Installation & Updates]
Hello, congratulations for this amazing job.

I'm having the same problem as described in [issue #1520](https://github.com/kevinpapst/kimai2/issues/1520), but can't figure out the solution. I think it's related to headers, but I need some help to solve this.
I also got a 405 "Method not allowed."

**Logfile**
[2021-12-11 02:02:16] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException: "No route found for "GET /api/timesheets/2/restart": Method Not Allowed (Allow: PATCH)" at /vendor/symfony/http-kernel/EventListener/RouterListener.php line 140

**Additional context**
 - Kimai version: 1.16.5 
 - PHP version: 7.4.26

**Comment by kevinpapst** [comment]
I'd say that there is an issue with the javascript. If javascript does not trigger, the browser will treat the link as link and use GET to open it. The javascript changes the request method to PATCH. Check your browser/javascript console what is happening.
Probably a javascript/privacy extension?

**Comment by Tertuliano** [comment]
Hi, thanks for the quick response.
I believe it's related with my configuration environment, perhaps .htacess and CSP, since I tried the demo with the same browser and everything it's working.
Also I check my extensions in Google Chrome, and nothing related to javascript. 

In issue #1520, the user WeirdCircunstances said he was able to fix the problem with his apache configuration, but the solution was not clear, I am trying to understand what is the procedure.

Can you get some ideas in that issue solution in order to solve this particular behavior?

Thanks in advanced.

**Reply by kevinpapst** [reply]
I guess he removed the Content-Security rules, not sure though.
If you find the solution, please share it for the next person experiencing this issue.
